tox.ini

```diff
@@ -12,8 +12,12 @@ commands = pylint -rn --rcfile={toxinidir}/pylintrc --load-plugins=pylint.extens
 
 [testenv:formatting]
 basepython = python3
-deps = black==18.6b4
-commands = black --check --exclude "functional|input|test/extension|test/regrtest_data|test/data" pylint
+deps =
+    black==19.3b0
+    isort==4.3.15
+commands =
+    black --check --exclude "functional|input|test/extension|test/regrtest_data|test/data" pylint
+    isort -rc pylint/ --check-only
```

Comment:
@PCManticore I added a command for isort here, so I believe there is no need to change the `travis.yml`, let me know if I'm wrong.

Reply:
Yes, that's fine!